README.md

```diff
@@ -51,13 +52,14 @@ You can also choose one of the presets provided by the ROS2 Gem to create a LiDA
 
 ## Requirements
 
-- [**Runtime requirements** of the Robotec GPU Lidar](https://github.com/RobotecAI/RobotecGPULidar#runtime-requirements).
+- [**Runtime requirements
+  ** of the Robotec GPU Lidar](https://github.com/RobotecAI/RobotecGPULidar#runtime-requirements).
```

Comment:
Please check rich diff - bold does not seem for format properly with this line break. Same thing happens in lines 130-131 below.
![image](https://github.com/user-attachments/assets/ee2d37a6-0241-44e9-baf2-cb96357c85fc)